### ISSUE TITLE
Angular 14 - infinite loop of Zone tasks 

### ISSUE THREAD

**Issue by uhlirradek** [tolgee/tolgee-js#3114]
Tolgee initialization starts an infinite loop of Zone.js tasks being processed, like hundereds per second. Everything was done properly following Installation steps. 

The main problem is that we had some events attached to ngZone.onStable() for positioning elements and Angular application freezes with that.

Debugging using ngDoCheck on AppComponent.ts and console logging Zone.currentTask shows the following (there is a huge amount of messages like this because change detection cycle is triggered in infinite loop)
`
runCount: 0,
source: "Promise.then",
type: "microTask",
_state: "notScheduled",
_zone: Zone {_parent: Zone, _name: 'angular', _properties: {…}, _zoneDelegate: ZoneDelegate}
`

Can you help me determine whats going wrong? 

When im not using Tolgee or Tolgee is not initialized properly (bad URL, bad API key etc.) the problem is not happening.

Thanks

**Comment by strejcradis** [comment]
Maybe I found a solution. Cloning ngx-tolgee package to my project and doing a small changes maybe fixed my problem. 

Wrapping part of code in TranslateService into runOutsideAngular fixes the problem:
Before: 
`   
this.runPromise = this.tolgee.run();
`

After:
`
this.zone.runOutsideAngular(()=> {
  this.runPromise = this.tolgee.run();
})
`

**Comment by JanCizmar** [comment]
Hello! Thanks a lot for clarification. We are currently working on the new `@tolgee/core` and `@tolgee/ngx`, so I will try to reflect this change and test it.

**Comment by JanCizmar** [comment]
It would be also nice if you can provide some minimal example to reproduce the issue. So I can see how you position the elements :)

**Comment by JanCizmar** [comment]
Anyway, I tried to run it this way: 
```
this.zone.runOutsideAngular(()=> { this.runPromise = this.tolgee.run(); })
```
And fixed the updating issues for the new `@tolgee/ngx` integration. However, I am not sure if this really helps in your case. 

I've created a prerelease version `4.9.4-rc.6a51a7a.0`. So you can try it with that. There is slightly different API, which is not yet documented. So you can use [testapp in the updated branch](https://github.com/tolgee/tolgee-js/tree/jancizmar/new-toogee-core/testapps/ngx/projects/sampleapp) as an inspiration.

**Comment by JanCizmar** [comment]
Hey! It's fixed in [5.0.0-alpha.1](https://www.npmjs.com/package/@tolgee/ngx/v/5.0.0-alpha.1). Tf you want to try, here are the docs: https://deploy-preview-313--tolgee-docs-preview.netlify.app/js-sdk/using_with_angular/installation

**Comment by JanCizmar** [comment]
It's fixed in v5. :)
